Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -888,6 +891,34 @@ extension SwiftLanguageServer {
       }
     }
   }
+
+  func executeCommand(_ req: Request<ExecuteCommandRequest>) {
```

Comment:
It's outdated, but the actual request is here:
https://github.com/apple/sourcekit-lsp/pull/70/files#diff-6175f3a32db6757aaeb5b49a98bab346R966